nytdiff.py

```diff
@@ -6,9 +6,12 @@
 import json
 import logging
 import os
+import shutil
 import sys
 import time
 
+from tempfile import TemporaryDirectory
```

Comment:
I don't think that should be necessary -- the `tempfile` package has been part of the Python core since 2.x, and `TemporaryDirectory` has been there since Python 3.2.

Reply:
I think I could add `python_version>=3.2` in requirements.txt to enforce that? But 3.2 has been EOL since 2016 and several of the other dependencies already enforce Python 3.8 or greater. Happy to take whichever approach you prefer :-)

Reply:
You're right, sorry. Searching I found a pypi link and I didn't check it was in the stdlib.